### PR TITLE
Allow check-runtime to fail

### PR DESCRIPTION
**Pull request by ddorgan**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -78,7 +79,7 @@ check-line-width:
     - /^[0-9]+$/
   script:
     - ./scripts/gitlab/check_line_width.sh
-  allow_failure:                   false
+  allow_failure:                   true
```
Comment (tomusdrw) [inline .gitlab-ci.yml]: Should we allow this to fail too?

Comment (bkchr) [reply .gitlab-ci.yml]: Yes and that is done :D

Comment (tomusdrw) [reply .gitlab-ci.yml]: Yeah, I just started the discussion IF that's actually a good change. I quite liked the long lines check, maybe we should just disable it in toml files, as there are no easy ways to overcome them sometimes, but in general it was pretty beneficial imho.

Comment (bkchr) [reply .gitlab-ci.yml]: The check is still running. It is just allowed to fail. We just had some hex string that needed to be written in one line.

**Pull request merged**